roles/openshift_health_checker/openshift_checks/fluentd_config.py

```diff
@@ -0,0 +1,157 @@
+"""
+Module for performing checks on a Fluentd logging deployment configuration
+"""
+
+import json
+
+from openshift_checks import OpenShiftCheck, get_var, OpenShiftCheckException
+from openshift_checks.logging.logging import LoggingCheck
+
+
+class FluentdConfig(OpenShiftCheck):
```

Comment:
Out of curiosity, why isn't this a subclass of the `LoggingCheck`? Perhaps we could avoid e.g. detecting whether logging is enabled and determining the logging namespace in two different files.